portfolio/src/main/java/com/google/sps/servlets/PageVisitsServlet.java

```diff
@@ -0,0 +1,98 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.Calendar;
+import java.util.List;
+
+@WebServlet("/visit")
+public class PageVisitsServlet extends HttpServlet {
+
+  private static final String VISITS_TODAY = "Visits Today";
+
+  private static final String VISITS = "visits";
+  private static final String DATE = "date";
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    incrementPageVisit(datastore);
+  }
+
+  private void incrementPageVisit(DatastoreService datastore) {
+    LocalDate today = LocalDate.now(ZoneOffset.UTC);
+    Entity todayVisitsEntity = getPageVisitsEntity(datastore, today);
+    long visitsToday = (long) todayVisitsEntity.getProperty(VISITS);
+    todayVisitsEntity.setProperty(VISITS, visitsToday + 1);
+
+    datastore.put(todayVisitsEntity);
+  }
+
+  private Entity createVisitsEntity() {
+    long timestamp = System.currentTimeMillis();
+    Calendar calendar = Calendar.getInstance();
+    LocalDate today = LocalDate.now(ZoneOffset.UTC);
+    Entity visitsEntity = new Entity(VISITS_TODAY);
+
+    visitsEntity.setProperty(DATE, today.toString());
+    visitsEntity.setProperty("dayOfWeek", calendar.get(Calendar.DAY_OF_WEEK));
```

Comment:
Nit: If you're using constants for some keys, consider using constants for all keys to make it consistent.

Reply:
I did this in [an upcoming pull request](https://github.com/josh8551021/step/pull/11).